From 6cdf8c4efa073eac7d5f9894329e2d07743c2955 Mon Sep 17 00:00:00 2001
From: Michael Roth <mdroth@linux.vnet.ibm.com>
Date: Thu, 25 Jun 2020 13:08:54 -0500
Subject: [PATCH] Update version for 4.2.1 release


Origin: upstream, https://git.qemu.org/?p=qemu.git;a=commit;h=6cdf8c4efa
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1891877
Last-Update: 2020-08-19

---
 VERSION | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/VERSION b/VERSION
index 6aba2b245a..fae6e3d04b 100644
--- a/VERSION
+++ b/VERSION
@@ -1 +1 @@
-4.2.0
+4.2.1
-- 
2.28.0

